bin/sdk-cli.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+/** @format */
+
+const chalk = require( 'chalk' );
+const execSync = require( 'child_process' ).execSync;
+const path = require( 'path' );
+
+const buildBlockScript = path.resolve( __dirname, 'create-scripts/block.js' );
+const crossEnvShell = path.resolve( __dirname, '../node_modules/.bin/cross-env-shell' );
+let args = process.argv;
+
+// Remove `node` and SDK script
+args.splice(0, 2);
+
+// First argument should always be the task name
+const task = args.shift();
+
+if ( ! args.length || task !== 'build-block' ) {
+	console.log( chalk.red( 'usage: npx calypso-gutenberg-sdk build-block <block>' ) );
+	process.exit(1);
+}
+
+execSync( `${ crossEnvShell } SKIP_FLAG_IMAGES=true node ${ buildBlockScript } ${ args.join( ' ' ) }`, {
```

Comment:
we're kind of working around the system here - `exec` sets `ENV` key/value pairs for us

```js
execSync( `node ${buildBlockScript} ${ args.join( ' ' ) }`, {
	env: {
		SKIP_FLAG_IMAGES: true,
	},
	shell: true,
	stdio: 'inherit',
} )
```

at this point of passing args though why not use `spawn` instead?

```js
spawnSync( 'node', [ buildBlockScript, ...args ], {
	env: {
		...process.env, // is this needed?
		SKIP_FLAG_IMAGES: true,
	},
	shell: true,
	stdio: 'inherit',
} );
```